Bio/__init__.py

```diff
@@ -125,7 +125,7 @@ class BiopythonExperimentalWarning(BiopythonWarning):
         " missing compiled C extensions. We recommend that you"
         " try running your code from outside the source tree."
         " If you are outside the source tree then you have a"
-        " setup.py file in an unexpected directory: {}.".format(_parent_dir),
+        " setup.py file in an unexpected directory: " + _parent_dir,
```

Comment:
could (optionally) be an f-string

Reply:
It could - honestly unsure in a case like this which is faster, or which is clearer.